lib/rex/parser/arguments.rb

```diff
@@ -70,26 +84,54 @@ def parse(args, &_block)
       def usage
         txt = ["\nOPTIONS:\n"]
 
-        fmt.sort.each do |entry|
-          fmtspec, val = entry
+        fmt.sort_by { |option| option.first.first.downcase }.each do |fmtspec, val|
```

Comment:
It's not immediately clear what `first.first` is here 👀 

I haven't checked this code, is this the same as this? 👀 

```suggestion
        fmt.sort_by { |key, _metadata| key.to_s.downcase }.each do |fmtspec, val|
```